docs/use/manual_storage_scaling.md

```diff
@@ -0,0 +1,66 @@
+# Manual storage scaling
+
+You can increase the storage capacity of your **S3-compatible** storage through manual storage scaling. This feature enables you to adapt to rising data demands, providing more flexibility in managing growing database workloads while ensuring system stability and data security.
+
+## Addressing the fixed Disk size constraint
+
+While you can adjust CPU and memory allocations for your databases, the disk size is fixed after the initial database provisioning. This enhancement allows you to manage expanding database workloads effectively while maintaining system stability and data security.
+
+In the Kubernetes environment used by Percona Everest, storage is managed using **Persistent Volumes (PVs)** and **Persistent Volume Claims (PVCs)**. Manual scaling involves adjusting the underlying PV resources to meet the demands specified in the PVC.
+
+For detailed information on PVs and PVCs, refer to the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/){:target="_blank"}.
+
+## Prerequisites
+
+- When scaling storage manually for a database managed by Percona Everest, ensure that the `StorageClass` used by the database's **PersistentVolumeClaim (PVC)** supports **volume expansion**.
+
+    !!! note
+        In Kubernetes, manual disk scaling only works if the associated `StorageClass` has the following setting:
+
+        ```sh
+        allowVolumeExpansion: true
+        ```
+
+    To verify if your storage class allows for volume expansion, execute the following command:
+
+    ```sh
+    kubectl get storageclass
+    kubectl describe storageclass <your-storage-class>
+    ```
+
+    ??? example "Expected output"
+        ```
+            apiVersion: storage.k8s.io/v1
+        kind: StorageClass
+        metadata:
+            name: expandable-storage
+            provisioner: kubernetes.io/aws-ebs
+            allowVolumeExpansion: true
+        ```
```

Comment:
Maybe the `kubectl get storageclass` command is enough, it gives the following output
```suggestion
    kubectl get storageclass
    ```

    ??? example "Expected output"
        ```
NAME                   PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
expandable-storage     kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   true                   42m

        ```
```

Reply:
@rasika-chivate Unresolving this comment since I don't think the above suggestion was applied